api-mobile/.gitignore

```diff
@@ -10,3 +10,5 @@ dist
 coverage
 
 .env
+env.local
```

Comment:
I don't think we want to ignore this file

Reply:
Yeah.. was thinking of this.
The problem is that it now contains credentials.

Reply:
Real ones.

Reply:
does it? I think thats just the default for a default postgres setup?

Reply:
The credentials we use in openshift are different

Reply:
All of the `**/env_config/*` files are only used in local setups, they aren't used in openshift for any of the real deployments

Reply:
But doesn't the make process always overwrite .env with env.local?

Reply:
The instructions in the readme say to copy the credentials into it.

Reply:
K... I'll just reverse this as well. I just put the credentials into my shell config.

Reply:
Its a bit confusing, but long story kind of short:

The main purpose of the makefile is just to facilitate the local docker setup.  
When you run `make local` it runs all of the commands needed to build/run docker locally.

The `make setup-local` and `env.local` were entirely just there as a bit of convenience when running the app locally. The only real difference between `env.local` and `env.docker` is the `DB_HOST` variable.

But, I've just realized that the api-mobile code never actually reads from the `.env` file.  So currently `make setup-local` doesn't really help with anything. But that being said, I am adding a tiny bit of code to api-mobile that will actually read the `.env`.  And then you will be able to run `make setup-local` and then edit the `.env` as needed to match your local postgres username/password, and then it should be happy.

Reply:
Sounds good @NickPhura